python/cudf/cudf/_lib/nvtext/subword_tokenize.pyx

```diff
@@ -0,0 +1,61 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+
+from libcpp cimport bool
+from libcpp.memory cimport unique_ptr
+from libcpp.string cimport string
+from libc.stdint cimport uint32_t
+from libc.stdint cimport uintptr_t
+
+from cudf._lib.move cimport move
+from cudf._lib.cpp.column.column cimport column
+from cudf._lib.cpp.column.column_view cimport column_view
+from cudf._lib.cpp.nvtext.subword_tokenize cimport (
+    subword_tokenize as cpp_subword_tokenize,
+    tokenizer_result as cpp_tokenizer_result,
+    move as tr_move
+)
+from cudf._lib.column cimport Column
+
+
+def subword_tokenize(
+    Column strings,
+    hash_file,
+    max_sequence_length=64,
+    stride=48,
+    do_lower=True,
+    do_truncate=False,
+    max_num_strings=100,
+    max_num_chars=100000,
+    max_rows_tensor=500
```

Comment:
We can type these directly in the Cython definition which also does dtype validation for us nicely:
```suggestion
    string hash_file,
    uint32_t max_sequence_length=64,
    uint32_t stride=48,
    bool do_lower=True,
    bool do_truncate=False,
    uint32_t max_num_strings=100,
    uint32_t max_num_chars=100000,
    uint32_t max_rows_tensor=500
```